pkg/grafana/types.go

```diff
@@ -44,12 +44,9 @@ func (d Datasource) buildJSONData() map[string]interface{} {
 
 func (d Datasource) buildSecureJSONData(organization Organization) map[string]string {
 	tenantIDs := organization.TenantIDs
-	if d.UID == mimirDatasourceUID {
-		// We do not support multi-tenancy for Mimir yet
-		tenantIDs = common.DefaultReadTenants
-	} else if d.UID == mimirAlertmanagerDatasourceUID {
-		// Alertmanager does not support multi-tenancy
-		tenantIDs = []string{common.DefaultWriteTenant}
```

Comment:
What's up with Alertmanager ?

Reply:
It will use the tenant in the organization CR now which is giantswarm